api/next.pb.txt

```diff
@@ -881,6 +881,13 @@ file {
       type: TYPE_STRING
       json_name: "variant"
     }
+    field {
+      name: "OSVersion"
+      number: 4
+      label: LABEL_OPTIONAL
+      type: TYPE_STRING
+      json_name: "OSVersion"
```

Comment:
```suggestion
      json_name: "osversion"
```

Reply:
this file was auto generated by "make protos"

Reply:
should I set the field name for json explicitly in platform.proto?

Reply:
yes I think that is where it should be modified not here

Reply:
Let me now if the new changes pushed look ok.